requirements/requirements-python3.8.txt

```diff
@@ -0,0 +1,370 @@
+# Editable install with no version control (apache-airflow==2.0.0.dev0)
```

Comment:
Somewhere there should be a checksum file generated for this to match -- look _somewhere_ in scripts/ci/

Reply:
It will all work fine as long the CI image will build.

Reply:
Shouldn't the precommithooks take care of this, i.e. it should be included in this PR?

Reply:
No. it takes to long. You ned to run `./breeze generate-requirements --python 3.8` manually. It takes far too long for pre-commits. But when the image builds and the requirements are wrong you get clear error and instructions "please run  `./breeze generate-requirements --python 3.8` `.

Reply:
Ah, gotcha.

Reply:
We need the checksum file though, right?

Reply:
Yep. It is automatically generated by "generate-requirements" command.

Reply:
@potiuk I seem to have done something incorrectly, because I'm seeing an error when `check-apache-license` runs. Could you take a look at the steps I've taken and let me know if anything seems incorrect?

Error: `Caused by: java.io.FileNotFoundException: /opt/airflow/bin/python` - I presume the inclusion of `/opt/airflow` means that it's trying to use the CI mount point, not the local ones?

Here's what I've done (locally).
1. Set `ARG DEPENDENCIES_EPOCH_NUMBER="4"` in Dockerfile.ci (previously was 3)
2. Set `PIP DEPENDENCIES_EPOCH_NUMBER="4"` in Dockerfile.ci (previously was 3)
3. Set `AIRFLOW_CONTAINER_CI_OPTIMISED_BUILD="false"`
4. Ran `./breeze generate-requirements --python 3.8` locally
5. Ran `./breeze generate-requirements --python 3.7` locally
6. Ran `./breeze generate-requirements --python 3.6` locally

- Did steps 1 & 2 so that CI wouldn't try to read the non-existent requirements-3.8 file from remote.
- Did step 3 so that the local `generate-requirements` command wouldn't use the remote master to build requirements lists.
- Did steps 5 and 6 because one of the pre-commit checks indicated that I needed to rebuild all requirements files after modifying setup.py.